src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -131,6 +165,9 @@ export class DictionaryModal extends React.Component {
                       placeholder="English"
                       onChange={this.onChange}
                     >
+                      {isEditingDictionary &&
+                        this.props.preSelectDefaultLocale()
```

Comment:
From the implementation of `preSelectDefaultLocale`:
```
preSelectDefaultLocale = () => {
  const { dictionary } = this.props;
  for (let i = 0; i < languages.length; i++) { // eslint-disable-line
   if (languages[i].value === dictionary.default_locale) {
   return (<option value={languages[i].value} selected>
   {languages[i].label}
   </option>);
  }
 }
 return null;
}
```
There is a possiblity of the function returning a `null` value, what then happens if this function returns `null`? Is the view not going to be affected?

Reply:
No, it is not going to be affected. Tested and confirmed.